test/README.md

```diff
@@ -2,18 +2,52 @@
 
 Thanks for writing tests! Here's a quick run-down on our current setup.
 
-## Tools we use
+## Getting started
+
+1. add a test to `packages/*/src/TheUnitInQuestion/TheUnitInQuestion.test.js` or `packages/*/test/`
+2. run `yarn test:watch`
+3. Implement the tested behavior
+4. Open a PR once the test passes or you want somebody to review your work
 
-Please familiarise yourself with these if you plan on contributing! :+1:
+## Tools we use
 
-- [mocha](https://github.com/mochajs/mocha)
-- [enzyme](https://github.com/airbnb/enzyme)
-- [jsdom](https://github.com/tmpvar/jsdom)
-- [karma](https://github.com/karma-runner/karma)
-- [chai](https://github.com/chaijs/chai)
-- [sinon](https://github.com/sinonjs/sinon)
-- [docker](https://github.com/docker/docker)
+- [@testing-library/react](https://testing-library.com/docs/react-testing-library/intro)
+- [chai](https://www.chaijs.com/)
+- [sinon](https://sinonjs.org/)
+- [mocha](https://mochajs.org/)
+- [karma](https://karma-runner.github.io/latest/index.html)
+- [enzyme](https://airbnb.io/enzyme/) (old tests only)
 - [vrtest](https://github.com/nathanmarks/vrtest)
+- [docker](https://docs.docker.com/)
+- [jsdom](https://github.com/jsdom/jsdom)
+
+## Writing Tests
+
+For all unit tests, please use the return value from `test/utils/createClientRender`.
+It prepares the test suite and returns a function with the same interface as
+[`render` from `@testing-library/react`](https://testing-library.com/docs/react-testing-library/api#render).
+
+For new test please use `expect` from the BDD testing approach. Prefer to use as expressive [matchers](https://www.chaijs.com/api/bdd/) as possible. This keeps
+the tests readable and more importantly the message if they fail as descriptive as possible.
+
+In addition to the core matchers from `chai` we also use matchers from [`chai-dom`](https://github.com/nathanboktae/chai-dom#readme).
+
+Deciding where to put a test is (like naming things) a hard problem:
+* when in doubt put the new test case directly in the unit test for that component e.g. `material-ui/src/Button/Button.test.js`
+* if you test interaction requiring multiple components from the library create a
```

Comment:
‘If your test interaction requires multiple components’?

Reply:
:thinking: I like yours better since it's more obvious what to do. Current wording focuses on what the interaction requires which isn't obvious. Using integration tests whenever the test i.e. what's inside `it('"works"', () => {})` uses multiple components is more obvious.

Reply:
Changed to 
> If your test requires multiple components from the library create a new integration test.

`interaction` doesn't add much value anyway.